libs/azure-iot-middleware-freertos/ports/coreHTTP/azure_iot_core_http.c

```diff
@@ -136,7 +136,7 @@ AzureIoTHTTPResult_t AzureIoTHTTP_Request( AzureIoTHTTPHandle_t xHTTPHandle,
         char headerbuffer[ 32 ] = { 0 };
         int valueLength = snprintf( headerbuffer, sizeof( headerbuffer ), "bytes=%d-%d", lRangeStart, lRangeEnd );
 
-        xHttpLibraryStatus = HTTPClient_AddHeader( &xHTTPHandle->xRequestHeaders, "x-ms-range", sizeof( "x-ms-range" ) - 1,
+        xHttpLibraryStatus = HTTPClient_AddHeader( &xHTTPHandle->xRequestHeaders, "Range", sizeof( "Range" ) - 1,
```

Comment:
We can replace this with `HTTPClient_AddRangeHeader()`.

https://github.com/FreeRTOS/coreHTTP/blob/ff33d4645807d36e314ce074597a15b70782969e/source/include/core_http_client.h#L715-L718

Reply:
No need for a temporary buffer

Reply:
I'll make that change.

Reply:
Done. And it works great. Thanks.